ts_src/payments/p2tr.ts

```diff
@@ -35,6 +34,9 @@ export function p2tr(a: Payment, opts?: PaymentOpts): Payment {
     throw new TypeError('Not enough data');
   opts = Object.assign({ validate: true }, opts || {});
 
+  if (!opts.eccLib) throw new Error('ECC Library is missing for p2tr.');
```

Comment:
nit: minor actionability improvement for this error message (and similar one in `p2tr_ns`)
```suggestion
  if (!opts.eccLib) throw new Error('ECC Library is required for p2tr.');
```